### ISSUE TITLE
Add option to show given Range in model dev-utils

### ISSUE THREAD

**Issue by oskarwrobel** [ckeditor/ckeditor5-engine#759]
Model `getData()` shows only Selection Range, but we can easily add an option to pass any Range to show (eg Marker Range).

**Comment by Reinmar** [comment]
For that you have `stringify()`. You pass any element and any range you want.

**Comment by oskarwrobel** [comment]
Hm... You are right.
I'm wondering what for we have both `getData` and `stringify`? `getData` is a wrapper for `stringify` that helps to get root element, but also limits its API.

**Comment by Reinmar** [comment]
`getData()` is just a shorthand. It's similar to how we had it in CKE4.

**Comment by pjasiun** [comment]
There is `options` parameter in `getData`. You can set there that there is no selection, so why you can not set there that it should show different selection?

In this case, it would be simpler to use `getData` which automatically gets root from the document.

And, to be honest, I also have no Idea why we need both, there could be just one: `getData( documentOrNode, options = {} )`

The only real difference is that the first one takes the document and the second one any node. If we want to keep this separation we should keep the same API:

```
getData( document, options = {} )
stringify( node, options = {} )
```

**Comment by pjasiun** [comment]
But I think we should have one function, keep it simple, at the end of the day this is a tool for devs, it should have simple and intuitive API to avoid questions like this.

**Comment by Reinmar** [comment]
I don't like in what direction it goes. I think that the current API makes some sense and it's convenient.

E.g. you have to have `setData()`, because it does more than `parse()`. So if you have `setData()`, then `getData()` makes sense too.

As for the params – I like being able to call `stringify( elem, range )` now and it should not become `options.range` because it doesn't have to be a range (can be a position or selection) and it be less convenient to use. So if this function needs options, then it should be a 3rd param: `stringify( elem, range, options )`, just like `setData( editor, data, options )`.

tl;dr: The current API is ok and let's not break it. If we had nice API docs *online*, then you would be able to find all the functions easily and there would be no confusion.

**Comment by oskarwrobel** [comment]
Ok, so `stringify` is enough.
